tests/helpers/WebView.ts

```diff
@@ -1,36 +1,115 @@
-import type { Context } from '@wdio/protocols';
-
 export const CONTEXT_REF = {
```

Comment:
@christian-bromann and @erwinheitzman 

If you have time, can you please check if you can follow what I am trying to explain here? Webviews are complex and I want to make sure users understand them a bit better.

Reply:
Very insightful, the documentation makes sense to me. I wonder if we can move some of it into the project docs. It would be cool to have a section on `Mobile Testing` and add these type of docs in them.